db/migrate/20190523124109_update_qualification_names.rb

```diff
@@ -0,0 +1,25 @@
+class UpdateQualificationNames < ActiveRecord::Migration
+  NAMES =
+    [
+      {
+        old: 'Chartered Financial Planner',
+        new: 'Chartered Financial Planner - accredited by the Chartered Insurance Institute / Personal Finance Society'
+      },
+      {
+        old: 'Certified Financial Planner',
+        new: 'Certified Financial Planner - accredited by the Chartered Institute of Securities and Investments'
+      }
+    ]
+
+  def up
+    NAMES.each do |name|
+      Qualification.where(name: name[:old]).first!.update(name: name[:new])
```

Comment:
Isn't it worth banging the update? How else would we realise it actually failed otherwise?